### DISCUSSION TITLE
Hide directories which do not exist on all merged remotes?

### DISCUSSION THREAD

**Discussion by ghost** [trapexit/mergerfs#1054] [Q&A]
Is it possible using mergerfs to hide directories which do not exist on all merged remotes?

**Comment by trapexit** [comment]
No

What's the usecase?

**Reply by ghost** [reply]
I have an issue with Lidarr where its behaviour scans the complete directory rather than just the artists directories added to it. More details on the issue here: https://www.reddit.com/r/Lidarr/comments/w59tx1/only_scanned_added_artists_rather_than_complete/

To get around this, I was looking to merge my rclone mounted album collection with Lidarr’s local post-processing directory. Directories from rclone which match those located in the post-processing directory should only be visible.

**Reply by trapexit** [reply]
It's plausible but very niche and a bit clunky to do.
